build-system/compile/closure-compile.js

```diff
@@ -88,7 +94,7 @@ function runClosure(outputFilename, options, flags, srcFiles) {
   return new Promise((resolve, reject) => {
     vinylFs
       .src(srcFiles, {base: getBabelOutputDir()})
-      .pipe(initializeClosure(flags))
+      .pipe(initializeClosure(flags, options))
```

Comment:
companion change to suggestion above
```suggestion
      .pipe(initializeClosure(flags, options.logger))
```